crates/relayer/src/chain/ckb/utils.rs

```diff
@@ -47,14 +47,14 @@ where
                 LightClientError::missing_last_block_id(height),
             ));
         }
-        is_creation = false;
+        // is_mmr_empty = false;
     }
 
-    if is_creation != onchain_packed_client_opt.is_none() {
-        return Err(Error::send_tx(
-            "storage and the chain state is not same".to_owned(),
-        ));
-    }
+    // if is_mmr_empty != onchain_packed_client_opt.is_none() {
+    //     return Err(Error::send_tx(
+    //         "storage and the chain state is not same".to_owned(),
+    //     ));
+    // }
```

Comment:
Remove these useless code.

Reply:
can leave this alone and erase later if truely make sure it's useless code